### PR TITLE
(docs): improved a11y-related text on icons page; added how-to for ad…

### PR DESCRIPTION
**Pull request by StommePoes**
…ding fonts

### CODE REVIEW

**Comment by StommePoes** [comment]
@aaronKaka, @parkerpearson does the text flow right, etc? 

I would ideally remove the aside tags as well since the content isn't separate at all-- these are more like "alert" style sections in an O'Reilly book (with like a little info icon, actually :)  and so I've reduced their h1's to h3's. We need to set a good example and not have multiple H1s on a page anyway-- only something like a modal dialog can maybe get away with it.

File: docs/elements/icons.md [hunk 1]
```diff
@@ -34,28 +34,46 @@ Add `pe-icon--{name}` to the icon element, where `{name}` is the icon name from
...
 <aside>
-  <h1 class="pe-title">Change the path to the icon font</h1>
+  <h3 class="pe-title">Change the path to the icon font</h3>
   <p>The default directory for the icon files is <code>../fonts/</code>. You may change the location and/or names of the files by overriding <code>$pe-icons-path</code> and <code>$pe-icons-name</code>, or by replacing the paths in the generated CSS files.</p>
 </aside>
 
+
 ## Making icons accessible
 
+The biggest accessibility problem with icons is that people don't tend to know what they are or what they mean unless they are associated with a textual label. Icons that are not just for presentation should provice a visible label near them. This concern regards all users but especially users with cognitive disabilities. This is very important if the icons are the main "text" of an interactive element (links, buttons).
```
Comment (aaronkaka) [inline docs/elements/icons.md]: "The major accessibility problem with icons is that users don't tend to know what they are or what they mean unless they are associated with a textual label. This particularly aids users with cognitive disabilities. This is especially important if the icons are the primary "text" of an interactive element (e.g. links, buttons)."

File: docs/elements/icons.md [hunk 2]
```diff
@@ -34,28 +34,46 @@ Add `pe-icon--{name}` to the icon element, where `{name}` is the icon name from
...
-</div>
+</button>
 {{/demo}}
+
+It's important to note that speech users (who interact with the page using voice commands) will not have access to the names of interactive elements whose text is visually hidden and only made available to screen readers. There are only a few icons where a user has a good chance of guessing the interactive element's name, such as "close" and "search", so it's important to show visible names of interactive elements as much as possible.
+
+
+## Adding new icons
+
+After the Design or UX teams have approved a new icon, you not only need to add the change to this page and to the css and test files, but you will also need to actually add the icon to the assets/fonts directory and a new *selection.json* file to misc/icomoon/.
```
Comment (aaronkaka) [inline docs/elements/icons.md]: "assets/fonts" -> "/assets/fonts"

**Comment by StommePoes** [comment]
You write much betters than I!

**Pull request merged**